game/menu/gamedisplay.gd

```diff
@@ -20,10 +24,10 @@ func setup(game_cfg: ConfigFile):
 
 	#setup of info Dialog
 	##setup buttons
-	$InfoButton.connect("pressed", $InfoDialog, "popup_centered_minsize", [Vector2(500, 250)])
-	var loadbtn = $InfoDialog.add_button("load")
-	loadbtn.connect("pressed", $InfoDialog, "hide")
-	loadbtn.connect("pressed", self, "_on_loadbutton_pressed")
+	_handle_error($InfoButton.connect("pressed", $InfoDialog, "popup_centered_minsize", [Vector2(500, 250)]))
+	_load_btn_dialog = $InfoDialog.add_button("load")
+	_handle_error(_load_btn_dialog.connect("pressed", $InfoDialog, "hide"))
+	_handle_error(_load_btn_dialog.connect("pressed", self, "_on_loadbutton_pressed"))
```

Comment:
These connect calls will never fail so _handle_error() is unnecessary.

Reply:
Not handling errors, just because is a pretty bad reason. Especially since that is not within a critical path.

Reply:
Not "Just because". These won't ever fail unless you break something. In that case they fail every time. There is no in-between. And the _handle_error() doesn't do anything connect doesn't do itself already.

Reply:
That's factually untrue. 

![grafik](https://user-images.githubusercontent.com/833043/173063163-c91966b1-1f4d-4c20-b007-8f22efc799bf.png)
![grafik](https://user-images.githubusercontent.com/833043/173062865-5b45252a-385c-4c7a-9d50-9fe62fbb75cb.png)

Also if an error occurs (e.g. pressed accidentially changed to pre0ssed) it prints something instead of silently failing:
![grafik](https://user-images.githubusercontent.com/833043/173063465-cfdd728c-964f-428b-887b-2c992a80ef49.png)

Is that likely to occur in this particular case? No of course not. But ignoring warnings because "ThEy cAnNoT FaIL" right now, assumes no-one is ever going to change the code. Will someone change the code in the future that is less skilled than you? Probably.

Reply:
That is a warning. It is a signal that something might not work as intended. But I specifically want this to happen so I'd tell Godot to ignore that warning.
If however, you use a wrong signal something like this will appear. 
![grafik](https://user-images.githubusercontent.com/60107996/173065847-3ae2f834-34ff-416c-a90c-a243608313ab.png)
Of course people will change this in the future, but reading debugger Errors and telling them apart from warnings is such a basic skill. I don't expect anyone seriously digging around in the core of the project to lack that skill.
And crucially if they don't get what the Debugger says they won't get what "Error 31" means.

If you really want to do this at least do it right.